src/script/lua_api/l_object.cpp

```diff
@@ -796,12 +701,14 @@ int ObjectRef::l_get_properties(lua_State *L)
 {
 	NO_MAP_LOCK_REQUIRED;
 	ObjectRef *ref = checkobject(L, 1);
-	ServerActiveObject *co = getobject(ref);
-	if (co == NULL)
+	ServerActiveObject *sao = getobject(ref);
+	if (sao == nullptr)
 		return 0;
-	ObjectProperties *prop = co->accessObjectProperties();
+
+	ObjectProperties *prop = sao->accessObjectProperties();
 	if (!prop)
```

Comment:
Since everything else is a `nullptr` now, I suppose it wouldn't hurt to make it consistent and have one more here that you missed.  But it's not important.